src/eko/runner/parts.py

```diff
@@ -150,9 +150,8 @@ def match(eko: EKO, recipe: Matching) -> Operator:
     op.compute()
 
     binfo = blowup_info(eko)
-    nf_match = op.nf - 1 if recipe.inverse else op.nf
     res, err = matching_condition.MatchingCondition.split_ad_to_evol_map(
-        op.op_members, nf_match, recipe.scale, **binfo
+        op.op_members, op.nf, recipe.scale, **binfo
```

Comment:
Actually, I was wandering if this could be a symptom of a deeper bug, something as the operator not multiplied with the correct order for backward evolution...